3.9/graphs-smart-graphs.md

```diff
@@ -83,159 +75,23 @@ replicated to each participating DB-Server, (weighted) graph traversals and
 DB-Server. This means a larger part of the query can be executed fully local
 whenever data from the SatelliteCollections is required.
 
-Benefits of Disjoint SmartGraphs
---------------------------------
+![Hybrid SmartGraphs](images/SmartGraphs_Hybrid.png)
+
+## Disjoint SmartGraphs
 
-Disjoint SmartGraphs are a specialized type of SmartGraphs. 
+Disjoint SmartGraphs are useful for use cases which have to deal with large hierarchical graphs, when you have clearly separated branches in your graph dataset. Disjoint SmartGraphs enable the automatic sharding of these branches and prohibits edges connecting them.
 
-In addition to the guaranteed sharding in SmartGraphs, a Disjoint SmartGraph
-prohibits edges between vertices with different `smartGraphAttribute` values.
+![Disjoint SmartGraphs](images/SmartGraphs-Disjoint.png)
 
-This ensures that graph traversals, shortest path, and k-shortest-paths queries
+_This ensures that graph traversals, shortest path, and k-shortest-paths queries
 can be executed locally on a DB-Server, achieving improved performance for
-these type of queries.
+these type of queries._
 
-Benefits of Hybrid Disjoint SmartGraphs
----------------------------------------
+## Hybrid Disjoint SmartGraphs
```

Comment:
The more I think about it the more I am in favour of dropping the hybrid wording. See above.